README.md

```diff
@@ -82,7 +82,19 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
-[linkedhashmap]: https://docs.oracle.com/javase/7/docs/api/java/util/LinkedHashMap.html
-[cats-sync]: https://typelevel.org/cats-effect/typeclasses/sync.html
+[scaffeine]: https://github.com/blemale/scaffeine
+[cats-sync]: https://typelevel.org/cats-effect/docs/typeclasses/sync
 
 [license]: http://www.apache.org/licenses/LICENSE-2.0
+
+[ci]: https://github.com/snowplow-incubator/scala-lru-map/actions?query=workflow%3ACI
+[ci-image]: https://github.com/snowplow-incubator/scala-lru-map/workflows/CI/badge.svg
+
+[releases]: https://maven-badges.herokuapp.com/maven-central/com.snowplowanalytics/scala-lru-map_2.13
+[release-image]: https://img.shields.io/maven-central/v/com.snowplowanalytics/scala-lru-map_2.13.svg
+
+[coveralls]: https://coveralls.io/github/snowplow/scala-lru-map?branch=master
+[coveralls-image]: https://coveralls.io/repos/github/snowplow/scala-lru-map/badge.svg?branch=master
```

Comment:
These badges are slightly wrong (noticed whilst trying to figure out why Coveralls is failing). Should be in `snowplow-incubator`.

```
[![Coverage Status](https://coveralls.io/repos/github/snowplow-incubator/scala-lru-map/badge.svg?branch=master)](https://coveralls.io/github/snowplow-incubator/scala-lru-map?branch=master)
```